cmd/manager/.gitignore

```diff
@@ -0,0 +1 @@
+manager
```

Comment:
I think we already ignored this binary in https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/.gitignore#L139 , it should be located in `bin/manager`?

Reply:
I don't know, but I saw this manager in my cmd/manager folder now while definitely I didn't create the manger by running 'go build', so not sure where it comes from...

Reply:
I usually use `make manager` in top dir.

Reply:
ok, let's close this ,if I meet this again I will reopen it , thanks for the commetns